docs/user-guide/configuration.md

```diff
@@ -146,6 +154,7 @@ stop_on_start = true
 stop_on_restart = false
 date_format = '%Y.%m.%d'
 time_format = '%H:%M:%S%z'
+matcher = regex
```

Comment:
Actually, I wonder why the `date_format` and `time_format` settings have quotes in this example. AFAICS they shouldn't.